en/register.md

```diff
@@ -57,6 +35,13 @@ At this point, your proxy will be assigned a Store ID, which can be retrieved
 with the status command. This will be used in later commands and to
 identify your proxy for support purposes.
 
+After successful registration it's advised to securely store the private key
+generated during the registration. This key is your proxy's identity. The key
```

Comment:
Maybe `s/during the registration/during the process/` to avoid repeating "registration".

Reply:
Done, thanks!